pkg/tfgen/examples_cache.go

```diff
@@ -167,55 +179,30 @@ func (*examplesCache) inferPlugins() map[string]map[string]string {
 	return rr
 }
 
-func (ec *examplesCache) inferBuildFileHashes() map[string]string {
-	candidates := []string{
-		"go.work",
-		"go.work.sum",
-	}
-	rr := map[string]string{}
-	for _, c := range candidates {
-		rr[c] = ec.filehash(c)
-	}
-	return rr
-}
-
 func (ec *examplesCache) inferSoftwareVersions() map[string]string {
-	used := []string{
-		"github.com/pulumi/pulumi/pkg/v3",
-		"github.com/pulumi/pulumi-terraform-bridge/v3",
-		"github.com/pulumi/pulumi-terraform-bridge/v3/pf",
+	v := map[string]string{}
+
+	bi, ok := debug.ReadBuildInfo()
```

Comment:
```suggestion
	buildInfo, ok := debug.ReadBuildInfo()
```

I read this as "bi" as in "binary" or "bilateral"